libgalois/include/katana/GraphTopology.h

```diff
@@ -844,7 +844,7 @@ class KATANA_EXPORT ProjectedPropGraphViewWrapper : public GraphTopologyTypes {
     return topo().original_to_projected_edge_id(eid);
   }
 
-  const PropertyGraph& property_graph() const noexcept { return *prop_graph_; }
+  const PropertyGraph* property_graph() const noexcept { return prop_graph_; }
```

Comment:
I'm a bit concerned about pointers (especially as member variables). Maybe it's time to start using shared pointers?

Reply:
Yeah, something like that.